packages/medusa/src/api-v2/admin/price-lists/query-config.ts

```diff
@@ -0,0 +1,63 @@
+import { prefixArrayItems } from "@medusajs/utils"
+import { defaultAdminProductsVariantFields } from "../products/query-config"
+
+export const defaultAdminPriceListFields = [
+  "id",
+  "created_at",
+  "deleted_at",
+  "description",
+  "ends_at",
+  "id",
+  "title",
+  "starts_at",
+  "status",
+  "type",
+  "updated_at",
+  "price_list_rules.price_list_rule_values.value",
+  "price_list_rules.rule_type.rule_attribute",
+  "price_set_money_amounts.money_amount.id",
+  "price_set_money_amounts.money_amount.currency_code",
+  "price_set_money_amounts.money_amount.amount",
+  "price_set_money_amounts.money_amount.min_quantity",
+  "price_set_money_amounts.money_amount.max_quantity",
+  "price_set_money_amounts.money_amount.created_at",
+  "price_set_money_amounts.money_amount.deleted_at",
+  "price_set_money_amounts.money_amount.updated_at",
+  "price_set_money_amounts.price_rules.value",
+  "price_set_money_amounts.price_rules.rule_type.rule_attribute",
+  ...prefixArrayItems(
+    defaultAdminProductsVariantFields,
+    "price_set_money_amounts.price_set.variant_link.variant."
+  ),
```

Comment:
No, we don't. I've made some changes here that aligns with this, will ping you when I've pushed.

Reply:
Nice, sounds good!

Reply:
@olivermrbl take a look at the latest commit, this is the prettiest I could make for now. 😄

The `fields` and `expands` is not friendly to the remote query approach, since the api has to know how the public interface is mapped to both the modules.